### PR TITLE
Dutch localization file (nl.json) 100% translated

### PR DESCRIPTION
**Pull request by renediepenbroek**
The complete UI is translated in Dutch (nl).

Best regards,

René

### CODE REVIEW

**Review by undsoft — APPROVED** [review]
Very nice. Thank you for your contribution.

**Comment by undsoft** [comment]
@renediepenbroek Tried to resolve conflicts myself, but doesn't look like I have permissions to edit your PR.
To resolve conflicts you can just pull master and accept your changes only (discarding master).

File: src/assets/i18n/nl.json [hunk 1]
```diff
@@ -1,4138 +1,4187 @@
...
-  "AuthVersion - optional - set to (1,2,3) if your auth URL has no version <a href=\"https://rclone.org/swift/#standard-options\" target=\"_blank\">(rclone documentation)</a>.": "AuthVersion - optional - set to (1,2,3) if your auth URL has no version <a href=\"https://rclone.org/swift/\\#standard-options\" target=\"_blank\">(rclone documentation)</a>.",
-  "Authentication URL for the server. This is the OS_AUTH_URL from an <a href=\"https://rclone.org/swift/#configuration-from-an-openstack-credentials-file\" target=\"_blank\">OpenStack credentials file</a>.": "Authentication URL for the server. This is the OS_AUTH_URL from an <a href=\"https://rclone.org/swift/\\#configuration-from-an-openstack-credentials-file\" target=\"_blank\">OpenStack credentials file</a>.",
-  "Change the default password to improve system security. The new password cannot contain a space or <b>#</b>.": "Change the default password to impro
```
Comment (undsoft) [inline src/assets/i18n/nl.json:3562]: `{maxVCPUs}` is a placeholder here and should not be changed to `{maxVCPUs's}`

Comment (undsoft) [inline src/assets/i18n/nl.json:4174]:
```
{n, meervoud, één {# docker image is} other {# docker images zijn }} bijgewerkt.
```
contains plural tokens and should be translated to something like this:
```
{n, plural, one {# docker image is} other {# docker images zijn }} bijgewerkt.
```

Comment (renediepenbroek) [reply src/assets/i18n/nl.json]: changed as I did with the other ones': 'doel' is changed to 'target

**Comment by renediepenbroek** [comment]
Hello Undsoft,

Please can you help me. I am a complete 'newcomer' especially with TrueNAS. I want to check my translated file on errors and syntax. I searched internet but cannot find where I should locate my translated file and/or if I have to adjust a kind of ini-file. You would help me very much. And by the way my translation if of course for public use.
Thanks in advance and best regards, 
René

**Comment by undsoft** [comment]
Hi,
There are some instructions on how to translate here https://github.com/truenas/webui/blob/master/README.md under `Translating Text to Other Languages`.
If there are any major issues with translation syntax, we'll be able to see it once you create a PR.

If you are asking about how to see you translations in TrueNAS interface, then they they will appear in next release (or next nightly) once merged. You can then select Dutch in TrueNAS settings.

**Pull request merged**